recipes/prismatic/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set version = "1.2.1" %}
+
+package:
+  name: prismatic
+  version: {{ version }}
+
+source:
+  url: https://github.com/prism-em/prismatic/archive/{{ version }}.tar.gz
+  sha256: 844f659575dd98999b7c4864e850a7ec735f297ab1fb6c2d2be91d7f9720fe70 
+  patches:
+    - cmakelists.patch
+
+build:
+  number: 0
+  # will skip windows & linux for the moment - until we've got OSX sorted...
+  skip: True             # [win]
+  skip: False            # [linux]
+  skip: False            # [osx]
+  features:
```

Comment:
```suggestion
```

Reply:
Yep - done (as part of a general tidy up commit)